non-buildable.libs-investigation.md

```diff
@@ -0,0 +1,10 @@
+| Lib Name                                | TS  | Build | Publish | Able to Build | Able to Publish |
```

Comment:
Delete this file

Reply:
Same as https://github.com/bikecoders/ngx-deploy-npm/pull/527#discussion_r1312432784